plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,114 @@
+package stomp
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net"
+
+	"github.com/go-stomp/stomp"
+
+	"github.com/influxdata/telegraf"
+	commontls "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+type STOMP struct {
+	Host      string          `toml:"host"`
+	Username  string          `toml:"username,omitempty"`
+	Password  string          `toml:"password,omitempty"`
```

Comment:
Please remove the `omitempty` setting
```suggestion
	Username  string          `toml:"username"`
	Password  string          `toml:"password"`
```

Reply:
I  need them as there are versions of  AMQ  supports integration without authentication.

Reply:
Well that's true, but in any case the value will be `nil` which means an empty string. `omitempty` only has an effect when serializing this struct which should never be the case actually.